Makefile

```diff
@@ -150,11 +150,11 @@ pkg: pkg-clean pkg-build pkg-deb # L=<lang> T=<type> [Z=gzip|xz] : Build fresh .
 
 ### Build and run application code
 
-frontend: # Compile frontend assets for production
-	npx webpack --mode=production
+frontend: # Compile and run frontend assets for production
```

Comment:
Why this interface change?

Reply:
This is not required, will revert it back.